src/world/message_server.cpp

```diff
@@ -110,12 +88,23 @@ void message_server_send(uint64 ipp, MSGSERVTYPE type, zmq::message_t* extra, zm
     }
 }
 
+std::string ipp_to_string(uint64 ipp)
+{
+    uint32  port = (uint32)((ipp & 0xFFFFFFFF00000000LL) >> 32);
```

Comment:
No need to mask here, binary shift doesn't act like a rotate. Though, I would leave comments here saying you're taking the top/bottom 32 bytes to do different things here

Reply:
Done